### DISCUSSION TITLE
question about integration

### DISCUSSION THREAD

**Discussion by DietmarHoch** [fabian-lauer/dbus-shelly-3em-smartmeter#7] [Q&A]
Hi,

the integration works so far and i think we have nearly the same Hardware configuration,
Shelly 3EM, Solax X1 and i plan to buy a victron Multiplus II 48V ( prices are to high at the moment, 1100Euro and up).
Please see my Screenshot, something looks diffrerent to yours in the solax integration.
<img width="1037" alt="Bildschirmfoto 2022-04-10 um 22 09 12" src="https://user-images.githubusercontent.com/43320118/162638341-a7420521-6860-42c0-9c6b-01d9204bb521.png">

1. i dont see sum of power in the left tile in the VRM portal, local works fine.
2. i dont see the "blue lines" for the Power flow.
3. and i dont know how i can change the config, that the solax inverter and the loads are on the left side.

i hope you can help.
email is gmx.de with a dot betwenn first and last name. (hope i can delete this later)
thanks,
Dietmar

**Comment by fabian-lauer** [comment]
Hi @DietmarHoch,

don't worry it looks the same for me in the VRM portal when the multiplus is switched off.
I am not sure about it how it should be and if other components (official once) will give another result.
